src/aiortc/rtcrtpreceiver.py

```diff
@@ -398,6 +399,14 @@ async def receive(self, parameters: RTCRtpReceiveParameters) -> None:
     def setTransport(self, transport: RTCDtlsTransport) -> None:
         self.__transport = transport
 
+    @property
+    def _enabled(self) -> bool:
+        return self.__enabled
+
+    @_enabled.setter
+    def _enabled(self, enabled: bool) -> None:
+        self.__enabled = enabled
+
```

Comment:
Same as for `RTCRtpSender`, just initialise `self._enabled = True` in the constructor, we don't need a getter / setter.